index.js

```diff
@@ -2,12 +2,19 @@
 
 const { writeFileSync } = require('fs')
 
+const getIgnoreList = () => {
+  const arg = process.argv.find(arg => arg.match('--ignore='))
+  if (!arg) return []
+  return arg.match(/--ignore=(.*)$/)[1].split(',')
```

Comment:
use `.shift()` or `.pop()` here instead of referencing the direct array index?

Reply:
How about this? I think it's more readable than using `match` again anyway.

https://github.com/ahmadnassri/node-updated/blob/573c30bfe0883365e24a83676a3e4fa7f499be86/index.js#L8